grep/plan.sh

```diff
@@ -1,15 +1,16 @@
+# shellcheck disable=SC2209
```

Comment:
This disables the check for the entire file and not just `pkg_name`.
To make it affect only the `pkg_name` line you need to make line 1 some comment like `# This comment line avoids shellcheck disabling checks for the entire file`